routes/auth.routes.js

```diff
@@ -0,0 +1,36 @@
+const { Router } = require("express");
+const router = new Router();
+
+const bcryptjs = require("bcryptjs");
+const saltRounds = 10;
+
+const User = require("../models/User.model");
+
+// GET route ==> to display the signup form to users
+router.get("/signup", (req, res) => res.render("auth/signup"));
```

Comment:
this view path does not exist in your project. If you wanted to render the file `signup.hbs`, this should have been `res.render('signup')`